bitnami/minio/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: minio
 sources:
   - https://github.com/bitnami/bitnami-docker-minio
   - https://min.io
-version: 11.2.15
+version: 11.2.16
```

Comment:
You are adding a new feature, I thinks increasing the minor version would be better

Reply:
Sure. Would be 11.3.0, right?

Reply:
Yes 👍